test/liquidation/liquidation.test.js

```diff
@@ -73,12 +81,18 @@ describe("Integration -- liquidation happy path", () => {
     })
 
     it("should purchase and withdraw signer bonds from deposit", async () => {
```

Comment:
Strictly speaking, this test is no longer asserting signer bonds were purchased. Maybe we should rename it to `should liquidate the deposit`?